### DISCUSSION TITLE
General Discussions on Redback Technologies and the Home Assistant Integration

### DISCUSSION THREAD

**Discussion by cabberley** [cabberley/HA_RedbackTech#4] [General]
General Discussions on Redback Technologies and the Home Assistant Integration

**Comment by Mark-Hetherington** [comment]
I've noticed that in the web portal there is now an API Access item under Support:
![image](https://github.com/user-attachments/assets/66485b5b-3105-48fe-beba-69a722328aac)

I'm not sure if this is only visible if someone has already requested API access, but it seems plausible to me that it might be visible to anyone. This might make it a bit easier to get an API key for new users. I'm not sure if it is worth putting that in the readme unless it is confirmed, but I'm not sure how else prospective users might come across it!

**Reply by cabberley** [reply]
Thanks for that! Haven't looked at the portal for a looooonnnggg time.
I will find out

**Reply by Mark-Hetherington** [reply]
Yes, I haven't needed to look at the portal since I got the integration all setup. Unfortunately our inverter has gone offline so I had to do some poking around.

**Reply by cabberley** [reply]
Mine does that a little too regularly for my liking as well.
Sometimes I can get away with just pushing the reset button other times I have had to shut the whole thing down, including the main breakers and cold start it.

**Reply by Mark-Hetherington** [reply]
Reset button? Now I'm curious! Ours does not have a reset button, but the "reset procedure" is to set the grid/inverter bypass switch to off. This effectively powers down the control board, but it also shuts off anything we have on our backup circuits. In our case, that is our modem, router, switches and so on. This means the internet might be still coming back up when the Redback control board is back.

The other option I've used previously is to unplug and replug the network cable at the router. 

Another thing I used to do was use the "Redback" (as opposed to "MyRedback") app. It had an option to connect locally to an inverter, and then you could toggle the "cloud connection" option which would usually get it back online. I'm getting an error in that app after it lists the inverter (with it's local IP) that prevents getting into the settings now though. I really should capture the traffic from the application though. Thinking about it, it probably opens up a way for me to connect to the Redback locally. From memory, when I was able to connect locally, it was at least able to read the power flow data.

**Reply by cabberley** [reply]
Maybe Depends on what you have. I have an st10000 battery hybrid the redback logo on the unit has a reset button behind it

**Reply by Mark-Hetherington** [reply]
![image](https://github.com/user-attachments/assets/20becd82-d131-42f0-9b01-ef0071901211)

:)
